oak_io/src/lib.rs

```diff
@@ -83,14 +83,105 @@ impl<L: Decodable, R: Decodable> Decodable for Either<L, R> {
     }
 }
 
+/// A wrapper struct that holds an init message, plus the handle of a channel from which to read
+/// command messages. This is useful for patterns in which an Oak node needs some data at
+/// initialization time, but then handles commands of a different type in an event-loop pattern.
```

Comment:
Does it mean that `Init` and `Commands` are now sent in separate channels?

Reply:
Yes, clarified. The point is that you can have a router node initialize the worker node with some information, and then send it a channel (coming from somewhere else) from which to read commands (e.g. gRPC invocations). I think this should help with the aggregator and trusted database pattern, let's try and use it and see if it makes things better.